Broadcast-Ext/OpenTok Live/OTBroadcastExtAudioDevice.m

```diff
@@ -47,6 +47,8 @@
 #endif
 
 static double kPreferredIOBufferDuration = 0.01;
+static double kToMicroSecond = 1000000;
```

Comment:
Why is the Broadcast Ext in this PR. Can we create a separate PR/ticket. Thanks

Reply:
I am removing those changes